e2e-tests/portal/cypress/integration/journeys/login/reset-password.spec.js

```diff
@@ -6,18 +6,32 @@ const MOCK_USERS = require('../../../fixtures/users');
 
 const { BANK1_MAKER1 } = MOCK_USERS;
 
-context('Reset password', () => {
+context('Password management screens', () => {
   beforeEach(() => {
     resetPassword.visitRequestEmail();
   });
 
-  context('Request password', () => {
+  context('Forgot / Reset password', () => {
     beforeEach(() => {
       resetPassword.visitRequestEmail();
     });
 
-    it('An incorrect email displays error message', () => {
-      resetPassword.emailInput().type('aaa');
+    it('Should have email address input, submit and cancel buttons on the page', () => {
+      cy.url().should('eq', relative('/reset-password'));
+      resetPassword.emailInput().should('exist');
+      resetPassword.submit().should('exist');
+      resetPassword.cancel().should('exist');
+    });
+
+    it('Enter an empty email address displays error message', () => {
```

Comment:
We should check here what the error message says when the user submits an empty form

Reply:
Updated.